src/main/java/frc/robot/Hardware.java

```diff
@@ -11,4 +11,8 @@ public class Hardware {
 
   // arm pivot [30-39]
   public static final int ARM_PIVOT_MOTOR_ID = 30;
+  // vision
+  public static final String PHOTON_IP = "10.24.12.51";
```

Comment:
This may conflict with an IP assigned by the field network. https://docs.wpilib.org/en/stable/docs/networking/networking-introduction/ip-configurations.html#on-the-field-static-configuration recommends 10.TE.AM.11